docs/products/postgresql/howto/create-read-replica.md

```diff
@@ -14,51 +19,83 @@ read-only queries to reduce the effect of slow queries on the primary
 node.
 :::
 
-## Create a replica
+## Prerequisites
+
+- Running Aiven for PostgreSQL service
+- Access the [Aiven Console](https://console.aiven.io/)
+- Optionally, for creating a read-only replica programmatically:
+  - [Aiven Operator for Kubernetes®](https://aiven.io/docs/tools/kubernetes)
+  - [Aiven API](https://api.aiven.io/doc/)
+  - [Aiven Provider for Terraform](https://registry.terraform.io/providers/aiven/aiven/latest/docs)
 
-To set up a remote replica:
-<!-- vale off -->
+:::important
+Creating a read-only replica programmatically, you can only use the Startup plan for the
+replica.
+:::
+
+## Create a replica
 
-1.  Log in to the [Aiven Console](https://console.aiven.io/).
-1.  Select the PostgreSQL instance for which you want to create a remote
-    replica.
-1.  In the **Overview** page of your service, select **Create replica**.
-1.  Enter a name for the remote replica and select the cloud provider,
-    region, and Aiven for PostgreSQL service plan that you want to use,
-    then select **Create**.
+<Tabs groupId="group1">
+<TabItem value="gui" label="Console" default>
 
-<!-- vale on -->
+1.  Log in to the [Aiven Console](https://console.aiven.io/), and go to your Aiven for
+    PostgreSQL service.
+1.  On the <ConsoleLabel name="overview"/> page of your service, click **Create replica**.
+1.  Enter a name for the remote replica, select the cloud provider, region, and service
+    plan to use.
+1.  Click **Create**.
 
 The read-only replica is created and added to the list of services in
-your project. The **Overview** page of the replica indicates the name of
-the primary service for the replica.
+your project. The <ConsoleLabel name="overview"/> page of the replica indicates the name
+of the primary service for the replica.
+
+</TabItem>
+<TabItem value="api" label="API">
+
+Use the [ServiceCreate](https://api.aiven.io/doc/#tag/Service/operation/ServiceCreate)
+endpoint and configure the `service_integrations` object so that:
+
+- `integration_type` is set to `read_replica`.
+- `source_service_name` is set as needed.
+
+</TabItem>
+<TabItem value="tf" label="Terraform">
+
+Use the
+[aiven_service_integration](https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/service_integration)
+resource. Set `integration_type` to `read_replica` and `source_service_name` as needed.
+
+</TabItem>
+<TabItem value="k8s" label="Kubernetes">
+
+[Create an Aiven for PostgreSQL read-only replica with K8s](https://aiven.github.io/aiven-operator/examples/postgresql.html#create-a-postgresql-read-only-replica).
+
+</TabItem>
+</Tabs>
 
-Read-only replicas can be manually promoted to become the master
-database if the need arises. For more complex high availability and
-failover scenarios check the
+Read-only replicas can be manually promoted to become the master database. For more high
```

Comment:
```suggestion
Read-only replicas can be manually promoted to become the primary database. For more high
```
Should we use primary instead of master?